debug/debugging.rst

```diff
@@ -56,5 +56,52 @@ by removing the call to ``loadClassCache()``::
     session.
 
 Some IDEs do not like the fact that some classes are stored in different
-locations. To avoid problems, you can tell your IDE to ignore the PHP cache
-file.
+locations. To avoid problems, you can either tell your IDE to ignore the PHP
+cache files, or you can change the extension used by Symfony for these files::
+
+    $kernel->loadClassCache('classes', '.php.cache');
+
+Useful debugging commands
+-------------------------
+
+When developing a large application, it can be hard to keep track of all the
+different services, routes and translations. Luckily, Symfony has some commands
+that can help you visualize and find the information.
+
+Information about the contents of the Symfony container, either for all
+public services or those matching a name, can be found using the commands:
+
+.. code-block:: bash
+
+    # Displays all public services
+    $ php bin/console debug:container
+
+    # Displays all public services that contain logger as part of their name
+    $ php bin/console debug:container logger
+
+To show all configured bundles, their class and their alias you can use the
+command:
+
+.. code-block:: bash
+
+    $ php bin/console debug:config
```

Comment:
I'm not sure it makes much sense to show the terminal examples, as they are just the name. What about using a definition list like:
```rst
When developing a large application, it can be hard to keep track of all the
different services, routes and translations. Luckily, Symfony has some commands
that can help you visualize and find the information.

``debug:container``
    To find information about the contents of the Symfony container, either for all
   public services, parameters or those matching a name.

``debug:config``
    To show all configured bundles, their class and their alias.

``debug:router``
    ...

...
```